src/test/java/cubecart/test/BaseTest.java

```diff
@@ -2,8 +2,56 @@
 
 import cubecart.utils.SessionUtil;
 import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
 
 public class BaseTest {
-    SessionUtil sessionUtil = SessionUtil.getInstance();
-    static Logger log = Logger.getLogger(BaseTest.class);
+
+    ChromeDriver driver;
+
+
+    public void setUp(){
+        System.out.println("Setting up resources for the test...");
+    }
+
+
+    public  void openBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://cubecartuat.unitedcoderapp.com/admin_d9nchr.php");
+    }
+
+
+
+    public  void login(String userName, String password) {
+        WebElement userNameField = driver.findElement(By.id("username"));
+        userNameField.sendKeys(userName);
+        WebElement passwordField = driver.findElement(By.id("password"));
+        passwordField.sendKeys(password);
+        WebElement loginButton = driver.findElement(By.id("login"));
+        loginButton.click();
```

Comment:
@Ilshat1207 after clicking the login button. how do you know you successfully logged in?  add another step here to assert the page header or title, so you are 100% sure that you are logged in successfully. otherwise no point running the test if login fails.